### PR TITLE
Fix checking of shell paths

### PR DESCRIPTION
**Pull request by SAtacker**
- Ref https://github.com/WasmEdge/WasmEdge/issues/2749

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: .github/workflows/test-python-install-script.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Setup Python env
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v3
```
Comment (hydai) [inline .github/workflows/test-python-install-script.yml:30]: What's the reason to downgrade the `setup-python` action?

File: utils/install.py [hunk 2]
```diff
@@ -1611,6 +1611,12 @@ def main(args):
         # Cleanup
         shutil.rmtree(TEMP_PATH)
 
+        if CONST_shell_config is None:
+            CONST_shell_config = "/DOES NOT EXIST;"
```
Comment (hydai) [inline utils/install.py:1615]: Should we make `/DOES NOT EXIST;` as a constant string?

**Review by hydai — CHANGES_REQUESTED** [review]

File: .github/workflows/test-python-install-script.yml (hunk 1 above)
Comment (SAtacker) [reply .github/workflows/test-python-install-script.yml:30]: Sorry, it was a bad rebase on my end.

**Comment by SAtacker** [comment]
@hydai what's with the macOS-arm environment ?

**Comment by hydai** [comment]
No need to install homebrew again. It should be installed on such env.

**Comment by SAtacker** [comment]
> No need to install homebrew again. It should be installed on such env.

@hydai please check https://github.com/WasmEdge/WasmEdge/actions/runs/5987695876/job/16242117229?pr=2752#step:3:6 it says brew is not found

**Comment by hydai** [comment]
@SAtacker 
Check this. I have no idea why the self-hosted macOS will have this issue. So we have to enable the brew env before executing `brew`.
https://github.com/WasmEdge/WasmEdge/pull/2728/files#diff-16c7e72c4bfd61962be81dc188052a8959cf9923fd34ca4388df7464c61712b1R35

**Pull request merged**